components/Dashboard/FormEdit.tsx

```diff
@@ -130,15 +130,23 @@ const FormEdit: React.FC<FormEditProps> = ({ reviewForm, onStepChange }) => {
               <FormControl>
                 <div className="relative mt-4 w-fit">
                   <input
+                    id="nameInput"
                     disabled={form.formState.isSubmitting}
                     className="w-[7.5rem] rounded bg-transparent px-1 py-1 text-xl font-bold outline-dashed outline-1 outline-transparent duration-100 hover:outline-gray-300 focus:outline-gray-300"
                     {...field}
                   />
-                  <Image
-                    src={edit_icon}
-                    alt="edit_icon"
-                    className="pointer-events-none absolute -right-5 bottom-2.5"
-                  />
+                    <Button
+                    type='button'
+                    style={{ backgroundColor: 'transparent' }}
+                    onClick={() => {
+                      const input = document.getElementById('nameInput');
+                      if (input) {
+                        input.focus();
+                      }
+                    }}
+                    >
```

Comment:
Hey @Anandateertha 😄
Can we change this to `useRef`?

Reply:
I am not able to use "useRef" because it is giving some kind of error saying "existing ref will be overwritten".